org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.json

```diff
@@ -0,0 +1,24 @@
+{
+  "id": "http://schemas.opentranslatorstothings.org/org.opent2t.sample.hub.superpopular#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "",
+  "title": "OpenT2T Hub",
+  "definitions": {
+    "org.opent2t.sample.hub.superpopular": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "ReadOnly, Gets the name of the hub."
+        },
+        "devices": {
+          "description": "ReadOnly, Gets the current set of devices associated with the hub."
+        }
```

Comment:
I'm not sure what to do here, should I create a custom type? would love your guidance/recommendation. the data that gets returned is: 
  "devices": [
    {
      "name": "Home Kitchen Thermostat (AB92)",
      "id": "152846",
      "openT2T": {
        "schema": "org.opent2t.sample.thermostat.superpopular",
        "translator": "com-wink-thermostat"
      }
    }

This is per the apiary doc. 

---
In reply to: [76334693](https://github.com/openT2T/translators/pull/75#discussion_r76334693) [](ancestors = 76334693,76154047)